models/Alert.js

```diff
@@ -0,0 +1,22 @@
+var mongoose = require('mongoose');
+
+var AlertSchema = new mongoose.Schema({
+  Id: Number
```

Comment:
The other schema use lowercase for their fields, we should probably be consistent

Reply:
I tried to follow the exact scheme Raph gave to me. 'ID' was a keyword so I replaced that with 'Id'.